### PR TITLE
Update the OSL page

### PR DESCRIPTION
**Pull request by kelnera**
Minor edits need to be made the OSL’s page

### CODE REVIEW

**Comment by kelnera** [comment]
@ramereth @Kennric @osu-cass/cass-media

File: content/pages/organization/units/osuosl.rst [hunk 1]
```diff
@@ -9,9 +9,9 @@ Open Source Lab
    :align: right
    :alt: Open Source Lab
 
-The `Open Source Lab (OSL)`_ has more than 10 years experience in supporting the
```
Comment (ramereth) [inline content/pages/organization/units/osuosl.rst]: Why are you removing this link? Please keep it as-is.

File: content/pages/organization/units/osuosl.rst [hunk 2]
```diff
@@ -29,7 +29,9 @@ impact Open Source projects including:
 * Gentoo
 
 In addition, we are also expanding our offering to include a variety of SaaS and
-PaaS solutions (e.g., Mailman, Drupal, Database as a service, OpenStack
-instances), to meet the needs of our customers.
+PaaS solutions (e.g., various web platforms, Mailman, Drupal, Database as a
+service, OpenStack instances), to meet the needs of our customers.
+
+For more information, visit our `website`_.
```
Comment (ramereth) [inline content/pages/organization/units/osuosl.rst]: You need to add the actual reference to the website URL :)

File: content/pages/organization/units/osuosl.rst (hunk 1 above)
Comment (kelnera) [inline content/pages/organization/units/osuosl.rst]: So you want two links to our website?

File: content/pages/organization/units/osuosl.rst (hunk 2 above)
Comment (kelnera) [inline content/pages/organization/units/osuosl.rst]: Oh I did forget that at the end. Do you want the url spelled out in the visible text as well?

File: content/pages/organization/units/osuosl.rst (hunk 1 above)
Comment (ramereth) [inline content/pages/organization/units/osuosl.rst]: Yup.

File: content/pages/organization/units/osuosl.rst (hunk 2 above)
Comment (ramereth) [inline content/pages/organization/units/osuosl.rst]: No, just linked.

Comment (kelnera) [inline content/pages/organization/units/osuosl.rst]: Alright, both issues have been fixed.

File: content/pages/organization/units/osuosl.rst [hunk 3]
```diff
@@ -29,7 +29,10 @@ impact Open Source projects including:
 * Gentoo
 
 In addition, we are also expanding our offering to include a variety of SaaS and
-PaaS solutions (e.g., Mailman, Drupal, Database as a service, OpenStack
-instances), to meet the needs of our customers.
+PaaS solutions (e.g., various web platforms, Mailman, Drupal, Database as a
```
Comment (ramereth) [inline content/pages/organization/units/osuosl.rst]: Please remove ``Drupal`` here too.

**Comment by ramereth** [comment]
:+1:

**Pull request merged**